config/initializers/devise.rb

```diff
@@ -236,7 +236,7 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :facebook, "1495604120735583", "4adef8998ba66e2643a8258f2c74a312", scope: 'email', info_fields: 'email, name, image'
+  config.omniauth :facebook, "1495604120735583", "4adef8998ba66e2643a8258f2c74a312", scope: 'email', info_fields: 'email, name'
```

Comment:
Good catch, thank you. Do you happen to know how we might hide this when deploying on production?